terraform/modules/front_door/load_balancer.tf

```diff
@@ -51,8 +53,15 @@ resource "aws_lb_listener" "https" {
   }
 }
 
+moved {

```

Comment:
This and the other moved blocks are here for clarity for the moment - they can be removed once we've done an apply on all existing environments (which I now have) but I was going to leave them in for the review process so it's clear what's happened.